.travis.yml

```diff
@@ -3,18 +3,24 @@ language: php
 php:
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+env:
+  - COMPOSER_OPTS=""
+  - COMPOSER_OPTS="--prefer-lowest"
+
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7
   fast_finish: true
 
-sudo: false
```

Comment:
I saw that in the flysystem .travis.yml file and copied it over. Does it not do anything?